stock_view_adj_rpl/views/stock_move_location_views.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Please change the file name to `stock_move_location.xml` and move it to `wizard` folder, respecting the structure of the original module.

Reply:
I got it. I have moved it.